Gemfile

```diff
@@ -30,7 +30,7 @@ gem 'therubyracer'
 
 # faster json
 # pinned on 2.17 because of an issue with 2.18: https://github.com/ohler55/oj/issues/325
-gem 'oj', '2.17.5'
+gem 'oj' #, '2.17.5'
```

Comment:
I can't remember doing this on purpose. The tests give "a Fabulous run " if the comment is not in place

295 runs, 1033 assertions, 0 failures, 0 errors, 0 skips

_[Original comment](https://github.ugent.be/unipept/unipept/pull/630#discussion_r15240) by @beardhatcode on Mon May 07 2018 at 16:33._